<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CryptoClient</class>
 <widget class="QWidget" name="CryptoClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1121</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CryptoClient</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>581</width>
     <height>371</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_Option">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>-10</y>
     <width>531</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_IP">
      <property name="title">
       <string>服务器IP信息（文件发送端口）</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="lineEdit_IP">
         <property name="text">
          <string>ip address</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLineEdit" name="lineEdit_ControlPort">
            <property name="text">
             <string>control port</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_FilePort">
            <property name="text">
             <string>filesend port</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_IP">
         <property name="text">
          <string>提交</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_File">
      <property name="title">
       <string>预备发送的文件位置</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEdit_File"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_File">
         <property name="text">
          <string>选择...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_Operate">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="pushButton_Connect">
         <property name="text">
          <string>建立连接</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_Disconnect">
         <property name="text">
          <string>关闭连接</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_PublicKeyGenerate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>641</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_DigitalSignature">
      <property name="title">
       <string>数字签名生成密钥</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QComboBox" name="comboBox_DigitalOption">
         <item>
          <property name="text">
           <string>ED25519（默认）</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_DigitalKeyGenerate">
         <property name="text">
          <string>生成...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_PublicConnect">
      <property name="title">
       <string>非对称通信密钥</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QComboBox" name="comboBox_PublicConnectOption">
         <item>
          <property name="text">
           <string>RSA-OAEP（默认）</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DH</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_PublicConnectGenerate">
         <property name="text">
          <string>生成...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_PrivateKey">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>360</y>
     <width>471</width>
     <height>331</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QGroupBox" name="groupBox_PrivateOption">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QComboBox" name="comboBox_PrivateOption">
         <item>
          <property name="text">
           <string>AES-128(无MAC)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GCM</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_RemoveFile">
         <property name="text">
          <string>将加密的中间文件移除</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_SendFile">
         <property name="text">
          <string>发送文件！</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_PublicKeyPosition">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>641</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QGroupBox" name="groupBox_DigitalKeyPosition">
      <property name="title">
       <string>数字签名密钥位置</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLineEdit" name="lineEdit_DigitalKeyPosition"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_DigitalKeyPosition">
         <property name="text">
          <string>选择...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_PublicConnectPosition">
      <property name="title">
       <string>非对称通信密钥位置</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLineEdit" name="lineEdit_PublicConnectPosition"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_PublicConnectPosition">
         <property name="text">
          <string>选择...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
